CHANGELOG.rst

```diff
@@ -0,0 +1,49 @@
+OMEGO version history
+=====================
+
+0.3.0 (February 2016)
+---------------------
+
+ * drops support for OMERO 5.1 and earlier
+ * remove `--ports` support. Use 5.2 properties.
```

Comment:
Some inconsistency in periods being here, not anywhere else. Also, the starts of the bullet points are sometimes capitalized (0.2.4 paragraph), sometimes not. cc @hflynn